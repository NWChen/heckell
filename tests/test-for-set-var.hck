let x: int set;
x = {5, 10, 13};

for i in x do
	print("\(i)");
end