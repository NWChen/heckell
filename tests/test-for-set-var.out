5
10
13
